platform/server/elasticsearch/call_api.ts

```diff
@@ -0,0 +1,37 @@
+import { get } from 'lodash';
+import { Client } from 'elasticsearch';
+
+type CallAPIOptions = { wrap401Errors?: boolean };
+type CallAPIClientParams = { [key: string]: any };
+
+export async function callAPI(
+  client: Client,
+  endpoint: string,
+  clientParams: CallAPIClientParams,
+  options: CallAPIOptions
+) {
+  const wrap401Errors = options.wrap401Errors !== false;
```

Comment:
Looked through kibana and x-pack, and can't find any usage of `wrap401Errors` (which means it's always `true`, from what I can tell). Maybe we could change this to:

```diff
diff --git a/platform/server/elasticsearch/call_api.ts b/platform/server/elasticsearch/call_api.ts
index 25420b4a7..cb1bfedc9 100644
--- a/platform/server/elasticsearch/call_api.ts
+++ b/platform/server/elasticsearch/call_api.ts
@@ -1,16 +1,15 @@
 import { get } from 'lodash';
 import { Client } from 'elasticsearch';
 
-type CallAPIOptions = { wrap401Errors?: boolean };
+type CallAPIOptions = { wrap401Errors: boolean };
 type CallAPIClientParams = { [key: string]: any };
 
 export async function callAPI(
   client: Client,
   endpoint: string,
   clientParams: CallAPIClientParams,
-  options: CallAPIOptions
+  options: CallAPIOptions = { wrap401Errors: true }
 ) {
-  const wrap401Errors = options.wrap401Errors !== false;
   const clientPath = endpoint.split('.');
   const api: any = get(client, clientPath);
 
@@ -24,7 +23,7 @@ export async function callAPI(
   try {
     return await api.call(apiContext, clientParams);
   } catch (err) {
-    if (wrap401Errors && err.statusCode === 401) {
+    if (options.wrap401Errors && err.statusCode === 401) {
       // TODO: decide on using homegrown error lib or boom
       // https://github.com/elastic/kibana/issues/12464
```

So the field becomes required, but it has a default value.

Reply:
@kjbekkelund i think wrap401Errors should still be optional. as in `type CallAPIOptions = { wrap401Errors?: boolean };` otherwise i agree

Reply:
Right? You said you couldn't find any usage of `wrap401Errors` (which means it's always `true` but that's beside the point), and that means if we make it required, then we have to add it _everywhere_ explicitly.

Reply:
I’m setting a default in that example, so it doesn’t have to be specified when calling the function

Reply:
```
options: CallAPIOptions = { wrap401Errors: true }
```